pkg/cmd/pulumi/state/state_unprotect.go

```diff
@@ -40,10 +40,12 @@ func newStateUnprotectCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "unprotect [resource URN]",
 		Short: "Unprotect resources in a stack's state",
-		Long: `Unprotect resource in a stack's state
+		Long: `Unprotect resources in a stack's state
 
 This command clears the 'protect' bit on one or more resources, allowing those resources to be deleted.
 
+To protect a resource, use ` + "`pulumi protect`" + `on the resource URN.
```

Comment:
I wouldn't advertise it, personally. In general this is not how you "protect" a resource.